src/CodeGenerator/FactoryGenerator.php

```diff
@@ -190,8 +190,17 @@ public function generate(string $class): string
 
         $filename = $this->buildFileName($class);
         $filepath = $this->outputDirectory . '/' . $filename;
-        $code     = strtr(
-            file_get_contents(self::TEMPLATE_FILE),
+        $template = file_get_contents(self::TEMPLATE_FILE);
+
+        if ($template === false) {
+            throw new RuntimeException(sprintf(
```

Comment:
@Ocramius 
Sweet christmas, this drops code coverage and reveals a smell. This code is never reached and really hard to test (almost impossible), since the file is shipped with this library. 
I'll replace this with an assert() so the test suite will bail if a change causes unexpected bahviour here and we don't carry uneccessary code.